checkout-ui-custom/src/controller/DeliverController.js

```diff
@@ -52,6 +53,17 @@ const DeliverController = (() => {
     }
   };
 
+  const RenderEditAddress = async (addressName) => {
+    const fields = 'id,receiverPhone,receiverName,number,companyBuilding,street,city,postalCode,state';
```

Comment:
We need to look at getting addresses via Index DB and then only using the fields relevant to us